src/syntax/Parser.js

```diff
@@ -2715,7 +2715,7 @@ export class Parser {
    *
    * @return {ParseTree}
    */
-  parseExpression_(expressionIn = Expression.IN) {
+  parseExpression_(expressionIn = Expression.NORMAL) {
```

Comment:
I realize this was a drive-by fix, but `Expression` is very overloaded in this file. Maybe the const type could be `InVariant` or `InExpressionVariant`.